docs/source/upgrade_notes.rst

```diff
@@ -47,6 +47,16 @@ Upgrade Notes
   workflow definition contains a task that is named ``continue``.
 * When installing packs from Exchange index ``st2 pack install <pack_name>`` will now download latest
   release from the remote repository, instead of using latest available git commit from master as before.
+* When upgrading an installation with the Community LDAP auth backend ``st2_auth_backend_ldap``
```

Comment:
Looks good!

Referencing it to exact Github repository https://github.com/StackStorm/st2-auth-backend-ldap is the only thing I can think of to avoid the confusion with the repos.